src/main/Config.h

```diff
@@ -28,6 +28,21 @@ struct HistoryArchiveConfiguration
 
 class Config : public std::enable_shared_from_this<Config>
 {
+    enum class ValidatorQuality : int
+    {
+        kLow = 0,
```

Comment:
Looks like you are adding 3rd convention for naming enum class entries.
Most of current ones use UPPER_CASE, one uses MixedCase (UpdateOfferResult).

Reply:
ah yes, good point